src/backend/looped/vehicle/bypass_speed.cpp

```diff
@@ -0,0 +1,26 @@
+#include "backend/looped_command.hpp"
+#include "gta/enums.hpp"
+#include "natives.hpp"
+
+namespace big
+{
+	class bypass_speed : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_tick() override
+		{
+			
+			ENTITY::SET_ENTITY_MAX_SPEED(self::veh, FLT_MAX);
+		}
+		
+		
+
+		virtual void on_disable() override
```

Comment:
Only a single newline between these functions.

Reply:
makes sense thank you again